src/resmom/catch_child.c

```diff
@@ -1968,8 +1965,7 @@ send_hellosvr(int stream)
 	return;
 
 err:
-	sprintf(log_buffer, "Failed to send IS_HELLOSVR message");
-	log_err(errno, msg_daemonname, log_buffer);
+	log_err(errno, msg_daemonname, "Failed to send IS_HELLOSVR message");
```

Comment:
how about `"Failed to send hello to server %s", <server ip:port>` ? (think of future, multi-host multi-server 😄  or failover in single server)